cpp/client/ct.cc

```diff
@@ -221,7 +221,7 @@ static bool PrecertChainToEntry(const cert_trans::PreCertChain& chain,
 
   entry->set_type(ct::PRECERT_ENTRY);
   string key_hash;
-  if (chain.CertAt(1)->SPKISha256Digest(&key_hash) != Cert::TRUE) {
+  if (chain.CertAt(1)->SPKISha256Digest(&key_hash) != util::Status::OK) {
```

Comment:
In a `CHECK`, comparing against `OK` is useful, because when it's not true, `CHECK_EQ` will dump what it got instead, but here, you could just use `chain.CertAt(1)->SPKISha256Digest(&key_hash).ok()`.